examples/resalloc/expected/man/resalloc-maint.1

```diff
@@ -39,9 +39,11 @@ The resource ID
 usage: resalloc-maint ticket-list [-h]
 
 .SH AUTHORS
-.B resalloc
-was written by Pavel Raiskup <praiskup@redhat.com>.
+.nf
+Pavel Raiskup <praiskup@redhat.com>
+.fi
+
 .SH DISTRIBUTION
-The latest version of resalloc may be downloaded from
+The latest version of resalloc\-maint may be downloaded from
```

Comment:
This was meant to be the name of the project, not the utility.

Reply:
fixed